modules/ROOT/pages/global-elements.adoc

```diff
@@ -0,0 +1,65 @@
+= Global Elements
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: studio, server, components, connectors, elements, palette, global elements, configuration elements
+
+A Global Element is a reusable object containing parameters that any number of elements in a flow can share. You reference a global element from Anypoint Connectors or components in your Mule application.
```

Comment:
Should this be presented as Global Element or global element? It is shown two different ways here, and throughout the Mule doc.